### ISSUE TITLE
Patches

### ISSUE THREAD

**Issue by kamalkech** [chapter-three/next-drupal#354]
Why still need to add those patch, Drupal not fixed yet on the current version ??

https://www.drupal.org/project/subrequests/issues/3049395
https://www.drupal.org/project/decoupled_router/issues/3111456

**Comment by shadcn** [comment]
Yes. These are not committed yet. For now, we need these to use subrequests and decoupled_router.

**Comment by bryangruneberg** [comment]
I'm setting up a demo, and I found for decoupled_router I needed to change the patch to `https://www.drupal.org/files/issues/2023-03-22/decouple_router-3111456-resolve-language-issue-63--get-translation.patch `
